t/080_super.t

```diff
@@ -16,8 +17,12 @@ my @SUPER = ();
     for (my $i = 0x0011_0000; $i < 0x7FFF_FFFF; $i += 0x200000) {
         push @SUPER, $i;
     }
-    for (my $i = 0x8000_0000; $i < 0xFFFF_FFFF; $i += 0x400000) {
-        push @SUPER, $i;
+
+    # Doesn't work on 32 bit.
+    if ($Config{ptrsize} == 8) {
```

Comment:
```suggestion
    if ($Config{ivsize} > 4) {
```

If `ivsize` is `4`, then maximal signed integer value is `2^31-1`. But `0x8000_0000` is `2^31`.

`ivsize` is compile time value and be 8 also for 32bit perl versions, when C compiler supports C99 long long type.